### ISSUE TITLE
Chromedriver fails to start

### ISSUE THREAD

**Issue by pradyut** [giggio/node-chromedriver#250]
Hi,

This is probably a me issue than an actual bug, but would love some help here.

I've installed chromedriver via WDIO. Essentially with this command https://webdriver.io/docs/gettingstarted.html#generate-configuration-file which fetches chromedriver. 

However if I try to run chromedriver it fails every time.  I've also tested on a coworkers machine. The version of chromedriver does match my Chrome version and also there's nothing running on those ports.

This is the error we get
`events.js:177
      throw er; // Unhandled 'error' event
      ^
Error: spawn /Users/lhoong/go.dev.bloomberg.com/SystemsCoreEngineering/bosun/client/node_modules/chromedriver/lib/chromedriver/chromedriver ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:264:19)
    at onErrorNT (internal/child_process.js:456:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:9)
Emitted 'error' event at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:270:12)
    at onErrorNT (internal/child_process.js:456:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:9) {
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn ' +
    '/Users/lhoong/go.dev.bloomberg.com/SystemsCoreEngineering/bosun/client/node_modules/chromedriver/lib/chromedriver/chromedriver',
  path: '/Users/lhoong/go.dev.bloomberg.com/SystemsCoreEngineering/bosun/client/node_modules/chromedriver/lib/chromedriver/chromedriver',
  spawnargs: []
}`

Any help would be appreciated,

Thanks,
Prad

**Comment by giggio** [comment]
I can't reproduce it. It seems you are on a Mac, right? Is the binary file there?

**Comment by pradyut** [comment]
The binary file is not there, could it be that the wdio install process does not install chromedriver with the auto download binary flag set to true?

**Comment by giggio** [comment]
I didn't know their config tool.
When you run `./node_modules/.bin/wdio config -y` you will see the install log, and then the output from the chromedriver module install. For example, mine says:

````
> chromedriver@80.0.1 install /p/rabiscos/webdriverio-test/node_modules/chromedriver
> node install.js

ChromeDriver binary exists. Validating...
ChromeDriver is already available at '/tmp/80.0.3987.16/chromedriver/chromedriver'.
Copying to target path /p/rabiscos/webdriverio-test/node_modules/chromedriver/lib/chromedriver
Fixing file permissions
Done. ChromeDriver binary available at /p/rabiscos/webdriverio-test/node_modules/chromedriver/lib/chromedriver/chromedriver
````

What does yours say?

**Comment by giggio** [comment]
Closing it as I can't repro. Let me know if you can find a way to repro it.
